src/release.js

```diff
@@ -614,9 +614,11 @@ export default function taoExtensionReleaseFactory(params = {}) {
             log.doing('Clean up the place');
 
             try {
-                await gitClient.deleteBranch(data.releasingBranch);   
+                await gitClient.deleteBranch(data.releasingBranch);
```

Comment:
There is a second possibility, using `if (gitClient.hasBranch)` and no `error.message` check, instead of try/catch.